### PR TITLE
Add auto translate 

### PR DESCRIPTION
**Pull request by richtier**
[this task](https://uktrade.atlassian.net/browse/ED-3481)

auto-translations

### CODE REVIEW

File: core/templates/core/google-cloud-credentials.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (richtier) [inline core/templates/core/google-cloud-credentials.html]: google translate api client requires the credentials to be provided as a file. I added a command `generate_google_translate_cerdentials` to generate it on deployment.

Comment (konradko) [reply core/templates/core/google-cloud-credentials.html]: could use

```
from django.template.loader import get_template_from_string

tpl = Template(get_template_from_string("My name is {{ my_name }}."))
```

File: core/models.py [hunk 2]
```diff
@@ -50,17 +50,20 @@ def draft_url(self):
...
         path = reverse(
             'wagtailadmin_pages:add',
             args=(self._meta.app_label, self._meta.model_name, 2)
         )
         return urljoin(environment_url, path) + '?' + querystring
 
+    def serialize_fields(self):
+        return {
+            key: value for key, value in model_to_dict(self).items()
+            if value is not None
```
Comment (sdonk) [inline core/models.py]: You can use `filter()`

Comment (richtier) [reply core/models.py]: will do as a follow up change

**Pull request merged**